src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -19,11 +19,7 @@
 import seedu.address.commons.util.CollectionUtil;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
-import seedu.address.model.person.Address;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.Name;
-import seedu.address.model.person.Person;
-import seedu.address.model.person.Phone;
+import seedu.address.model.person.*;
```

Comment:
I think it would be better to keep the single import statements rather than use a wildcard import